src/neynar-api/neynar-api-client.ts

```diff
@@ -1738,6 +1740,7 @@ export class NeynarAPIClient {
       parentUrl?: string;
       channelId?: string;
       embedUrl?: string;
+      embedType?: EmbedType[];
```

Comment:
This should be `embedTypes` cuz fetchFeed in src/neynar-api/v2/client.ts expects `options?.embedTypes`